gqlalchemy/query_builder.py

```diff
@@ -469,42 +469,51 @@ def from_(
     def where(self, item: str, operator: str, value: Any) -> "DeclarativeBase":
         """Creates a WHERE statement Cypher partial query."""
         separator = "" if operator == ":" else " "
-        self._query.append(
-            WhereConditionPartialQuery(
+        query = (
+            WhereConditionPartialQuery(WhereConditionConstants.WHERE, separator.join([item, operator, value]))
+            if operator == ":"
+            else WhereConditionPartialQuery(
                 WhereConditionConstants.WHERE, separator.join([item, operator, to_cypher_value(value)])
             )
         )
+        self._query.append(query)
 
         return self
 
     def and_where(self, item: str, operator: str, value: Any) -> "DeclarativeBase":
         """Creates a AND (expression) statement Cypher partial query."""
         separator = "" if operator == ":" else " "
-        self._query.append(
-            WhereConditionPartialQuery(
+        query = (
```

Comment:
Rewrite this if statement. The only part that is changing is the third parameter in join, so move if-else there.

Reply:
This is all different in new PR for WHERE clause, so please check there.

Reply:
Can we... Revert this, then?